erpnext/hr/doctype/attendance/attendance_list.js

```diff
@@ -1,6 +1,12 @@
 frappe.listview_settings['Attendance'] = {
 	add_fields: ["status", "attendance_date"],
-	get_indicator: function(doc) {
-		return [__(doc.status), doc.status=="Present" ? "green" : "darkgrey", "status,=," + doc.status];
+	get_indicator: function (doc) {
+		if (doc.status == "Present" || doc.status == "Work From Home") {
```

Comment:
Combine the conditions, like:

```javascript
if (["Present", "Work From Home"].includes(doc.status) {
	// do things
}
```

Which is similar to doing the `<value> in <iterable>` check in Python